data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -0,0 +1,144 @@
+import os
+import re
+import uuid
+
+from google.cloud import dataproc_v1 as dataproc
+from google.cloud import storage
+import pytest
+
+# Set global variables
+PROJECT = os.environ["GCLOUD_PROJECT"]
+DATAPROC_CLUSTER = f"clean-test-{uuid.uuid4()}"
+BUCKET_NAME = f"clean-test-code-{uuid.uuid4()}"
+CLUSTER_REGION = "us-east4"
```

Comment:
Let's try changing this to us-central1, which has a higher CPU quota in our project than us-east4